src/sys/reboot.rs

```diff
@@ -0,0 +1,50 @@
+//! Reboot/shutdown or enable/disable Ctrl-Alt-Delete.
+
+use {Errno, Error, Result};
+use libc;
+use void::Void;
+use std::mem::drop;
+
+/// How exactly should the system be rebooted.
+///
+/// See [`set_cad_enabled()`](fn.set_cad_enabled.html) for
+/// enabling/disabling Ctrl-Alt-Delete.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum RebootMode {
```

Comment:
Regarding the names, you understood me correctly.

I had a look at the Travis output and I dont understand it. The constants are `libc::c_int` in libc, which is `i32` hence an enumeration with #[repr(i32)] should work.